src/index.js

```diff
@@ -2,7 +2,13 @@
  * @typedef {object} CantoToolData
  * @description Canto Tool's input and output data format
  * @property {string} contentId — canto content id
- * @property {metaData} meta — fetched canto data
+ * @property {string} src - image URL
+ * @property {string} title - image title
+ * @property {string} alt - image content description
+ * @property {string} caption - editorial for image
+ * @property {string} credit - attribution for image
+ * @property {string} height - height of image in px
+ * @property {string} width - width of image in px
```

Comment:
If moving these to `CantoToolData`, we don't need the `metaData` object anymore. Please remove as cleanup.